cdap-examples/README.md

```diff
@@ -43,17 +43,17 @@ Each example comes with a Maven pom.xml file. To build, install Maven, and from
 
 ## Purchase
 
-- An app that uses scheduled MapReduce Workflows to read from one ObjectStore Dataset
+- An app that uses scheduled MapReduce program and Workflow to read from one ObjectStore Dataset
```

Comment:
We have this in several places... but we don't have a way to schedule MapReduce directly. I think the correct way to say this is: a scheduled Workflow that runs a MapReduce program to ..., or a scheduled Workflow with a MapReduce program to...